src/cascadia/Remoting/WindowManager.cpp

```diff
@@ -274,7 +274,6 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
             p->AssignID(givenID.value());
         }
 
-        // If the name wasn't specified, this will be an empty string.

```

Comment:
seems like an irrelevant change?

Reply:
Oh wait... is this no longer possible because `_WindowName` is now `Window {id}` from Peasant.cpp?

Reply:
nope that's my bad, this comment should have been left in